systems/metrics/api-gateway/pkg/rest/router.go

```diff
@@ -153,6 +153,11 @@ func (r *Router) init(f func(*gin.Context, string) error) {
 				info.Description = "Get metrics for a susbcriber. Response has Prometheus data format https://prometheus.io/docs/prometheus/latest/querying/api/#range-vectors"
 			}}, tonic.Handler(r.subscriberMetricHandler, http.StatusOK))
 
+		auth.GET("/sites/:site/orgs/:org/metrics/:metric", []fizz.OperationOption{
+				func(info *openapi.OperationInfo) {
+					info.Description = "Get metrics for a site. Response has Prometheus data format https://prometheus.io/docs/prometheus/latest/querying/api/#range-vectors"
+				}}, tonic.Handler(r.siteMetricHandler, http.StatusOK))
+	
```

Comment:
Shouldn't that be a websocket if its providing metrics. Also if its a metric it should be part of metrics system

Reply:
Will be done in another PR